tensorboard/components/vz_line_chart2/microbenchmark/renders_spec.ts

```diff
@@ -0,0 +1,372 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {benchmark, Size} from './spec.js';
+import {polymerFlush} from './polymer_util.js';
+
+const DATA_POINTS = {
+  sine1k: [...new Array(1000)].map((_, index) => ({
+    step: index,
+    wall_time: index * 1000,
+    scalar: Math.sin(index / (2 * Math.PI)),
+  })),
+  cosine1k: [...new Array(1000)].map((_, index) => ({
+    step: index,
+    wall_time: index * 1000,
+    scalar: Math.cos(index / (2 * Math.PI)),
+  })),
+  cosine100k: [...new Array(100000)].map((_, index) => ({
+    step: index,
+    wall_time: index * 1000,
+    scalar: Math.cos(index / (2 * Math.PI)),
+  })),
+};
+
+const FIVE_HUNDRED_1K_DATA_POINTS = [...new Array(500)].map((_, index) => {
```

Comment:
Below, never >100 elements of this array is used, so why create 500?

Reply:
`redraw: 500 lines of 1k points` uses it. I must be not getting your feedback. What do you mean?

Reply:
Ah. I missed that part.